adapters/adot/adot.go

```diff
@@ -60,6 +60,7 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 		Uri:     endpoint,
 		Body:    reqJSON,
 		Headers: headers,
+		ImpIDs:  []string{request.Imp[0].ID},
```

Comment:
Another example of just the first imp id being specificed, but there may be more in the request.

Reply:
modified